ckanext/ontario_theme/templates/external/scheming/package/read.html

```diff
@@ -17,7 +17,7 @@
       {{ h.render_markdown(h.scheming_language_text(pkg['notes_translated'])) }}
     </div>
   {% endif %}
-  <h3>{% trans %}For more information{% endtrans %} </h3>
+  <h2 class="ontario-h2">{% trans %}For more information{% endtrans %} </h2>
```

Comment:
`ontario-h2` can be removed because the h2 tags don't have any overrides

Reply:
done